docs/first_time_setup.md

```diff
@@ -7,50 +7,62 @@ Before starting, it's recommended that you run all of these steps [in VS Code an
+   7. In your MongoDB Atlas `promptu` project, go to _Deployment > Database_ and click on _Connect_ on your `promptu-db`. Select _Connect your application_ and copy the connection string up until (but not including!) the URL path and query strings. So if you're connection string looks like `mongodb+srv://promptu:<password>@promptu-db.p4jpncm.mongodb.net/?retryWrites=true&w=majority`, you only need `mongodb+srv://promptu:<password>@promptu-db.p4jpncm.mongodb.net`.
+   8. Replace `<password>` in the connection string from `3.7.` with the value secured from `3.6.` - keep this value secure!
+   9. Use the `flyctl` CLI tool to create a `PROMPTU_MONGODB_URL` secret with your MongoDB connection URL from `3.8.` with the following script - you have to be in the same directory as the each service's `fly.toml` (**note:** make sure you're running this from a safe environment or from within a script, as your secret will be preserved within your shell history if run directly in your shell environment):
+
+   ```sh
+   cd apps/feeder-api
+   flyctl secrets set --detach PROMPTU_MONGODB_URL="<secure value from 3.8. for the promptu-feeder user>"
+
+   cd ../../apps/db-updater
+   flyctl secrets set --detach PROMPTU_MONGODB_URL="<secure value from 3.8. for the promptu-db-updater user>"
+   ```
+
+4. **Scale Kafka memory**
+   1. Our application will need at least 2048MB (or 2GB) of memory to run. Before you run your Github workflows to deploy your applications for the first time, you have to run the following command to scale it's memory:
+   ```sh
+   cd apps/kafka
+   flyctl scale memory -a promptu-kafka-eds 2048
+   ```
+5. **Prepare Github Workflow**
+   1. In your forked Github repo, go to _Settings > Security > Secrets > Actions_ and create the following repository secrets:
+      - (**Sensitive**) `FLY_API_TOKEN` (value secured from step `2.4.`)
+   2. Update your `fly.toml` files to include the suffix you provided in step `3.4` (if you chose `paper_mache` as your suffix, then your app name will be `promptu-paper_mache` for the `ui` component and `promptu-api-paper_mache` for the `api` component)
+6. **Raise your first PR and merge it into `main` to build your infrastructure and deploy your apps** - now that all the scaffolding has been set up, it's time to add some bricks! Merging your first PR will deploy your application to fly.io for the first time! All subsequent PRs will not only deploy the latest changes to fly.io, but it will also update your infrastructure through Terraform Cloud. But we're not done yet - we still need to secure access to our DB!
```

Comment:
Should we go with eds/main for now? I'm also thinking that maybe even after the workshop, leaving the repo with two branches (we could make eds/main protected against deletion if it isn't) - would help people orienteer easier? Could feel overwhelming doing both of the setup at the same time for someone who is new to infrastructure.

Reply:
The `main` branch is more general to their setup and not specific to ours, but I'll make that more clear